ajax/libs/stacktable.js/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "stacktable.js",
+  "filename": "stacktable.min.js",
+  "version": "1.0.1",
+  "description": "jQuery plugin for stacking tables on small screens",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/johnpolacek/stacktable.js.git"
+  },
+  "keywords": [
+    "js",
+    "javascript",
+    "jquery",
+    "tables"
+  ],
+  "author": {
+    "name": "John Polacek",
+    "url": "http://www.johnpolacek.com"
+  },
+  "license": "MIT",
+  "homepage": "http://johnpolacek.github.com/stacktable.js/",
```

Comment:
https://johnpolacek.github.com/stacktable.js/ will also be redirect to http://johnpolacek.github.io/stacktable.js/.
I think https://johnpolacek.github.io/stacktable.js/ is better.